crypto/evp/asymcipher.c

```diff
@@ -159,18 +160,22 @@ static int evp_pkey_asym_cipher_init(EVP_PKEY_CTX *ctx, int operation,
         goto err;
     }
 
+    desc = cipher->description != NULL ? cipher->description
+        : "no asym cipher description by provider";
```

Comment:
This indentation looks bad to me. Also the text is IMO too bulky - I would at least remove the _by provider_ part.

Reply:
The indentation is fine according the the Emacs C mode and the `check-format.pl` tool, 
but anyway, after chopping the " by provider" part, the RHS is less wide and fits into the 100 char limit.